TheMET/TheMET/LoadingPlaceholderView/LoadingPlaceholderView.swift

```diff
@@ -27,22 +28,21 @@ class LoadingPlaceholderView: UIView {
     
     override init(frame: CGRect) {
         super.init(frame: frame)
+        self.container.translatesAutoresizingMaskIntoConstraints = false
         self.animationView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
There is no need to configure `translatesAutoresizingMaskIntoConstraints` for `animationView` and `textLabel` as they are put to stack-view